dnf/cli/format.py

```diff
@@ -15,8 +15,23 @@
 # Red Hat, Inc.
 
 from __future__ import unicode_literals
+from datetime import datetime
```

Comment:
Well, the line bellow, i.e.
  from dnf.pycomp import long
breaks code conventions in the same way - long is referenced only once...